content/docs/06-integrations/00-trident.md

```diff
@@ -0,0 +1,203 @@
+
+<br />
+
+#### Create a backend.  
+
+Trident supports multiple backends and depending your storage backend, you will need to choose a backend that fits your needs. You can find example driver manifests [here](https://github.com/NetApp/trident/tree/master/trident-installer/sample-input/backends-samples) and you will need to personalise them with your credentials and environment configurations. Here's an example with the ONTAP-NAS driver. 
+
+<br />
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backend-ontap-nas-secret
+  namespace: trident
+type: Opaque
+stringData:
+  username: [USERNAME]
+  password: [PASSWORD]
+---
+apiVersion: trident.netapp.io/v1
+kind: TridentBackendConfig
+metadata:
+  name: backend-ontap-nas
+  namespace: trident
+spec:
+  version: 1
+  storageDriverName: ontap-nas
+  managementLIF: [ONTAP_MANAGEMENT_LIF_IP]
+  dataLIF: [DATA_LIF_IP]
+  backendName: ontap-nas
+  autoExportCIDRs:
+  - [x.x.x.x/xx]
+  autoExportPolicy: true
+  svm: [SVM]
+  credentials: 
+    name: backend-ontap-nas-secret
+```
+
+<br />
+
+#### Create a storage class.  
+
+Kubernetes allows us to bind statically and/or dynamically provisioned volumes to our Pods. Statically provisioned volumes are manually created by a user and then referenced in a deployment. Astra Trident allows you to leverage your NetApp storage and 
+Before you can request dynamically provisioned volumes, you will need to create a storage class. Storage classes 
+
+<br />
+
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: trident-csi
+  namespace: trident
+provisioner: csi.trident.netapp.io
+parameters:
+  backendType: "ontap-nas"
+  csi.storage.k8s.io/fstype: ext4
+```
+
+<br />
+
+#### Provision your first Volume.  
+
+Here's an example of a basic Persistent Volume Claim (PVC). 
+
+<br />
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: trident-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: trident-csi
+```
+
+<br />
+
+#### Deploy your first Pod and mount the volume.  
+
+Here's an example of a simple Pod deployment. This Pod contains an NGINX container, with a claim to the previously created Persistent Volume Claim (PVC). Once deployed, the PVC with it's bound Persistent Volume (PV), will mount to the Pod and provide persistent storage to this application.    
+
+<br />
+
+```yaml
+kind: Pod
+apiVersion: v1
+metadata:
+  name: example-pv-pod
+  namespace: trident
+spec:
+  volumes:
+    - name: example-pv-vol
+      persistentVolumeClaim:
+       claimName: trident-pvc
+  containers:
+    - name: example-pv-pod
+      image: nginx
+      ports:
+        - containerPort: 80
+          name: "http-server"
+      volumeMounts:
+        - mountPath: "/usr/share/nginx/html"
+          name: example-pv-vol
+```
```

Comment:
Let's merge these into one heading called  ### Example Deployment.  In the intro, explain what the example configurations will do. You already have the text, it just needs to be massaged into an intro for a merged section.